lambda/health_package/metric-settings.json

```diff
@@ -1,4 +1,206 @@
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 599,
+    "Description": "The number of bytes successfully put to the Kinesis stream over the specified time period. This metric includes bytes from PutRecord and PutRecords operations. Minimum, Maximum, and Average statistics represent the bytes in a single put operation for the stream in the specified time period."
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "IncomingRecords",
+    "Statistic": "Average",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 0.99,
+    "Description": "The number of records successfully put to the Kinesis stream over the specified time period. This metric includes record counts from PutRecord and PutRecords operations. Minimum, Maximum, and Average statistics represent the records in a single put operation for the stream in the specified time period."
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "GetRecords.Success",
+    "Statistic": "Average",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 0.99,
+    "Description": "Alert on 1% failure"
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "GetRecords.Records",
+    "Statistic": "Average",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 50,
+    "Description": "The number of records retrieved from the shard, measured over the specified time period. Minimum, Maximum, and Average statistics represent the records in a single GetRecords operation for the stream in the specified time period."
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "GetRecords.Bytes",
+    "Statistic": "Average",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 35000,
+    "Description": "The number of bytes retrieved from the Kinesis stream, measured over the specified time period. Minimum, Maximum, and Average statistics represent the bytes in a single GetRecords operation for the stream in the specified time period."
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "GetRecords.IteratorAgeMilliseconds",
+    "Statistic": "Maximum",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 5000,
+    "Maximum": 1800000,
+    "Description": "Maximum 1/2 an hour"
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "GetRecords.Latency",
+    "Statistic": "Average",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 5,
+    "Maximum": 26,
```

Comment:
The way the code works is the threshold is calculated based on past behaviour and the Max/Min are there to contain that automatically derived threshold. According to the docs this is measured in ms. Are we really saying we'd always want to know if it were more than 26ms?

Reply:
The current trend over the past 2 weeks is how the automatic threshold is calculated in `generate_metric_alarms.py` we don't need to worry about that. We just need to set a min/max that we don't want that automatically calculated threshold to stray outside of.